### DISCUSSION TITLE
Who is using LinGoose?

### DISCUSSION THREAD

**Discussion by henomis** [henomis/lingoose#160] [General]
Are you using LinGoose in your project or are you planning to use it? Tell me more here 🚀

**Comment by akshaylb** [comment]
Really like this library till now @henomis!
Currently trying to use it for a local project. 
I really like the open interface definition and the ability to override LLM implementation if needed!

**Comment by zt9** [comment]
I'm using it to build a knowledge base from phone call transcripts that I have had with a bunch of team members.. since it's our conversation, it is very useful to be able to just ask a question about common problems, etc.

**Comment by astrada-c** [comment]
I really like the way this project is structured and that it's not a carbon copy of langchain, I'm using it to create specialised agents starting with a learning assistant. Ollama tools support would let me move entirely to this project so hopefully it makes it to the feature list.

**Comment by kmesiab** [comment]
I have created an SMS based copilot tool for therapists to enhance patient services, mental health analysis, highly personalized mental health care planning, and on demand interventions.   

I have tried several different Golang based LLM abstraction libraries with varying levels of quality.  This library has come a long way.
